### ISSUE TITLE
`rust-lld: error while loading shared libraries: libLLVM-7.so` on recent nightly

### ISSUE THREAD

**Issue by koute** [rust-lang/rust#53869]
Recent `wasm32-unknown-unknown` builds on my Travis started failing with the following error:

```
rust-lld: error while loading shared libraries: libLLVM-7.so: cannot open shared object file: No such file or directory
```

[rustc 1.30.0-nightly (02cb8f2a4 2018-08-29) failed](https://travis-ci.org/koute/stdweb/jobs/422776918)
[rustc 1.30.0-nightly (7061b2775 2018-08-28) worked fine](https://travis-ci.org/koute/stdweb/jobs/422182259)

**Comment by koute** [comment]
Looking at the list of commits between `7061b2775` and `02cb8f2a4` this commit look like a possible culprit: https://github.com/rust-lang/rust/commit/3cf6f0db1ab61ed33c585b156a0d5c41279f0810

**Comment by koute** [comment]
cc @michaelwoerister

**Comment by memoryruins** [comment]
Duplicate of #53813, #53849 
Fixed in #53828, which will be in the next nightly.

**Comment by koute** [comment]
Ah, my apologies! I searched for the issue, but I didn't look through closed issues. -_-
